packages/loader/container-loader/src/container.ts

```diff
@@ -401,7 +403,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return this._protocolHandler;
     }
 
-    private resumedOpProcessingAfterLoad = false;
+    private inboundQueuePausedFromInit = false;
```

Comment:
I think the value should flip here and everywhere it's used to reflect the new name 😬